templates/rust/common.tera

```diff
@@ -538,12 +689,20 @@ where
         }
     }
 
+    /// Get mask for bitfield, mask is unshifted and at the actual offset
```

Comment:
I think this should be removed `and at the actual offset`

Reply:
I put it there because I found "mask is unshifted" to be unclear. Does "unshifted" mean that it's at the LSB and not shifted into the actual position - or does "unshifted" mean that it's at the actual position and not shifted down. It depends on which assumption you are starting with.
Reworded now to "the mask is shifted to the position of the bitfield" - that doesn't repeat itself and should be clear?